ogre/src/OgreRenderEngine.cc

```diff
@@ -19,6 +19,16 @@
 #if !defined(__APPLE__) && !defined(_WIN32)
 # include <X11/Xlib.h>
 # include <X11/Xutil.h>
+
+#if !defined(kronos_intptr_t) || !defined(khronos_intptr_t)
```

Comment:
I am afraid this only checks if `kronos_intptr_t` is defined as a C preprocessor macro, it will not check if `kronos_intptr_t` is a typedef.

Reply:
Ah, good call.